### PR TITLE
azurerm_log_analytics_cluster - update the validation for size_gb

### PR DESCRIPTION
**Pull request by neil-yechenwei**
This PR is to update the validation for `size_gb`.

--- PASS: TestAccLogAnalyticsCluster_resize (1932.09s)
--- PASS: TestAccLogAnalyticsCluster_basic (2118.79s)
--- PASS: TestAccLogAnalyticsCluster_requiresImport (2191.80s)

### CODE REVIEW

File: website/docs/r/log_analytics_cluster.html.markdown [hunk 1]
```diff
@@ -49,7 +49,7 @@ The following arguments are supported:
 
 * `size_gb` - (Optional) The capacity of the Log Analytics Cluster specified in GB/day. Defaults to 1000.
```
Comment (WodansSon) [inline website/docs/r/log_analytics_cluster.html.markdown:50]:
```suggestion
* `size_gb` - (Optional) The capacity of the Log Analytics Cluster is specified in GB/day. Possible values include `500`, `1000`, `2000` or `5000`. Defaults to `1000`.
```

File: website/docs/r/log_analytics_cluster.html.markdown [hunk 2]
```diff
@@ -49,7 +49,7 @@ The following arguments are supported:
...
 
-~> **NOTE:** The `size_gb` can be in the range of 1000 to 3000 GB per day and must be in steps of 100 GB. For `size_gb` levels higher than 3000 GB per day, please contact your Microsoft contact to enable it.
+~> **NOTE:** The cluster capacity must start at 500 GB and can be set to 1000, 2000 or 5000 GB/day. For more information on cluster costs, see [Dedicate clusters](https://docs.microsoft.com/en-us/azure/azure-monitor/logs/cost-logs#dedicated-clusters).
```
Comment (WodansSon) [inline website/docs/r/log_analytics_cluster.html.markdown:52]:
```suggestion
~> **NOTE:** For more information on cluster costs and capacity please see the [product documentation](https://docs.microsoft.com/azure/azure-monitor/logs/cost-logs#dedicated-clusters).
```

**Review by WodansSon — CHANGES_REQUESTED** [review]
@neil-yechenwei, thank you so much for this PR... the changes look mostly good to me but I have a few suggestions on the wording in the documentation. Can you take a look and let me know what you think? 🚀

File: website/docs/r/log_analytics_cluster.html.markdown (hunk 1 above)
Comment (neil-yechenwei) [reply website/docs/r/log_analytics_cluster.html.markdown:50]: Updated

File: website/docs/r/log_analytics_cluster.html.markdown (hunk 2 above)
Comment (neil-yechenwei) [reply website/docs/r/log_analytics_cluster.html.markdown]: Updated

**Review by WodansSon — APPROVED** [review]
@neil-yechenwei, thanks for pushing those changes... this LGTM! 🚀

**Comment by mybayern1974** [comment]
This PR fixes #17787

**Comment by niks-singh** [comment]
Can you please provide the version number in which this change will be available to use. I do not see this change in [v3.16.0] in registry

**Review by stephybun — APPROVED** [review]
Thanks for this @neil-yechenwei and for the review @WodansSon. I went ahead and rebased this so that we can get this in, hope you don't mind.

**Pull request merged**